src/app_event.h

```diff
@@ -12,7 +12,17 @@
 
 class LEDWidget;
 
-enum class AppEventType : uint8_t { None = 0, Button, ButtonPushed, ButtonReleased, Timer, UpdateLedState };
+enum class AppEventType : uint8_t {
```

Comment:
Not sure if this change is needed. It now works without additional AppEvents.

Reply:
Then it is not needed :)